internal/integration/mysql_test.go

```diff
@@ -1406,12 +1408,12 @@ func (t *myTest) cmdCmpShow(ts *testscript.TestScript, _ bool, args []string) {
 		// Trim the "table_options" if it was not requested explicitly.
 		stmts = append(stmts, create[:strings.LastIndexByte(create, ')')+1])
 	}
-
 	// Check if there is a file prefixed by database version (1.sql and <version>/1.sql).
 	if _, err := os.Stat(ts.MkAbs(filepath.Join(t.version, fname))); err == nil {
 		fname = filepath.Join(t.version, fname)
 	}
 	t1, t2 := strings.Join(stmts, "\n"), ts.ReadFile(fname)
+	t1, t2 = reSpace.ReplaceAllString(t1, " "), reSpace.ReplaceAllString(t2, " ")
```

Comment:
Call you elaborate why it's needed, and why `TrimSpace` is not enough?

Reply:
Using `Replace` will edit also the table content (default values or expressions).